src/Symfony/Component/Translation/DataCollector/TranslationDataCollector.php

```diff
@@ -97,8 +97,13 @@ public function getLocale()
         return !empty($this->data['locale']) ? $this->data['locale'] : null;
     }
 
+    /**
+     * @deprecated
+     */
     public function getFallbackLocales()
     {
+        @trigger_error('getFallbackLocales() is deprecated since version 4.1 and will be removed in 5.0.', E_USER_DEPRECATED);
```

Comment:
deprecation is in 4.2, not in 4.1. We cannot deprecate it in 4.1 as it is already released since 4 months.

Reply:
@stof  tahnk you for your effort to teaching me! (i will work on this tomorrow 👍 )